pkg/event/queue.go

```diff
@@ -19,60 +19,85 @@ package event
 
 import (
 	"sync"
+
+	"github.com/optimizely/go-sdk/pkg/logging"
 )
 
 // Queue represents a queue
 type Queue interface {
-	Add(item interface{})
+	Add(item interface{}) // Should return a bool
```

Comment:
TODO?

Reply:
Yeah probably a TODO item, but unfortunately it would break the interface otherwise I would have done it here. Likely we'd have to define another interface and add backwards compatibility but wrapping any old implementations into the new one.